Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.py

```diff
@@ -0,0 +1,1203 @@
+
+        return int(fetch_days_ago.timestamp())  # type: ignore
+
+
+def incidents_to_import(
+    alerts: List[Dict[str, Any]],
+    last_run: Optional[str] = demisto.getLastRun().get("last_run"),
+    first_fetch: Optional[str] = demisto.params().get("first_fetch", "2 days")
+) -> List[Dict[str, Any]]:
+    """
+    Helper function to filter events that need to be imported.
+    It filters the events based on the `created_at` timestamp.
+    Function will only be called if the SecurityScorecard API returns more than one alert.
+
+    Args:
+        ``alerts``(``List[Dict[str, Any]]``): A list of alerts to sort through.
+        ``last_run``(``str``): UNIX timestamp when the fetch incidents had last run.
+        ``first_fetch`` (``str``): Configured 'First fetch' parameter
+    Returns:
+        ``List[Dict[str, Any]]``: Alerts to import
+    """
+
+    last_run = get_last_run(last_run=last_run, first_fetch=first_fetch)  # type: ignore
+
+    demisto.debug(f"Last run timestamp: {last_run}")
+
+    incidents_to_import: List[Dict[str, Any]] = []
+
+    # Check if there are more than 0 alerts
+    if alerts:
+
+        # The alerts are sorted by ascending date so last alert is the most recent
+        most_recent_alert = alerts[-1]
+
+        most_recent_alert_created_date = most_recent_alert.get("created_at")
+
+        most_recent_alert_timestamp = \
+            int(datetime.strptime(most_recent_alert_created_date, SECURITYSCORECARD_DATE_FORMAT).timestamp())  # type: ignore
+        demisto.debug(f"Setting last runtime as alert most recent timestamp: {most_recent_alert_timestamp}")
+        demisto.setLastRun({
+            'last_run': most_recent_alert_timestamp
+        })
+
+        for alert in alerts:
+
+            alert_created_at = alert.get("created_at")
+            alert_timestamp = int(datetime.strptime(alert_created_at, SECURITYSCORECARD_DATE_FORMAT).timestamp())  # type: ignore
+
+            alert_id = alert.get("id")
+            company_name: str = alert.get("company_name")  # type: ignore
+            change_type: str = alert.get("change_type")  # type: ignore
+
+            debug_msg = f"import alert '{alert_id}'? (last_run < alert_timestamp): {(last_run < alert_timestamp)}"  # type: ignore
+
+            demisto.debug(debug_msg)
+
+            if alert_timestamp > last_run:  # type: ignore
+                incident = {}
+                incident["name"] = f"{company_name} {change_type.replace('_', ' ').title()}"
+                incident["occurred"] = \
+                    datetime.strptime(alert_created_at, SECURITYSCORECARD_DATE_FORMAT).strftime(DATE_FORMAT)  # type: ignore
+                incident["rawJSON"] = json.dumps(alert)
+                incidents_to_import.append(incident)
+    # If there are no alerts then we can't use the most recent alert timestamp
+    # So we'll use now as the last run timestamp
+    else:
+        now = int(datetime.utcnow().timestamp())
```

Comment:
You can't do this, you will accidentally miss alerts. For the last run always use the latest alert time.

Reply:
fixed.